framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java

```diff
@@ -348,16 +349,38 @@ private static int getMissingCount(Map<ColumnList, Integer> map) {
   }
 
   private static int compareTo(ColumnList list1, ColumnList list2,
-      Map<Integer, String> columnIndexAndOrder, int start) {
```

Comment:
This looks good now. @rhou1 can you take a look at the readme file and see anything there need to be updated after this change. Specifically, when we construct queries with order by clause, what are the rules we should follow? Would be nice if you can give some example query in the readme. Thanks.